app/controllers/channel_works_controller.rb

```diff
@@ -3,6 +3,7 @@ class ChannelWorksController < ApplicationController
 
   def index(page: nil)
     @works = current_user.works.wanna_watch_and_watching
+              .published
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.